app/Http/Controllers/ManageUsersController.php

```diff
@@ -93,4 +94,23 @@ public function showPage(): View|RedirectResponse
             'title' => 'Manage Users'
         ]);
     }
+
+    public function ajaxFindUsers(): View
+    {
+        $config = Config::getInstance();
+
+        $search = $_GET['search'];
+        if ($config->get('CDASH_FULL_EMAIL_WHEN_ADDING_USER') == 1) {
+            $sql = "email=?";
+            $params = [$search];
+        } else {
+            $sql = "email LIKE '%' || ? || '%' OR firstname LIKE '%' || ? || '%' OR lastname LIKE '%' || ? || '%'";
```

Comment:
In MySQL the `||` characters _may_ be used for concatenation ( a certain sql_mode will do that but MYSQL defaults to an logical "OR" ) . Any search returns all users.  Can we add the "if postgres" that exists other places?

Reply:
MySQL and PostgreSQL both support the `CONCAT()` command, so I have replaced the `||` operator with the `CONCAT()` command.